src/i18n/en.js

```diff
@@ -26,6 +26,25 @@ export default function loadEn() {
         'We believe that mentorship should be accessible to all people regardless of location or financial standing. In pursuit of this goal we will provide a free and open source platform to facilitate mentorship connections. Our mission is to foster a greater sense of collaboration and inclusiveness in the technical industry by providing a platform to aid the mentorship process.',
       'home-mission-media-alt': 'Group of people',
       'slack-org': 'Slack Organization',
+      close: 'Close',
+      'privacy-policy': `Effective date: October 03, 2018 {{lineBreak}}
+        Coding Coach ("us", "we", or "our") operates the website and the Coding Coach mobile application (the "Service").{{lineBreak}}
+        This page informs you of our policies regarding the collection, use, and disclosure of personal data when you use our Service and the choices you have associated with that data. Our Privacy Policy for Coding Coach is managed through {{freePrivacyPolicy}}. {{lineBreak}}
+        We use your data to provide and improve the Service. By using the Service, you agree to the collection and use of information in accordance with this policy. Unless otherwise defined in this Privacy Policy, terms used in this Privacy Policy have the same meanings as in our Terms and Conditions.`,
+      'free-privacy-policy': 'Free Privacy Policy',
+      'cookies-policy': 'Some cookie policy',
+      "<0>What Are Cookies<0><1/><2/>As is common practice with almost all professional websites this site uses cookies, which are tiny files that are downloaded to your computer, to improve your experience. This page describes what information they gather, how we use it and why we sometimes need to store these cookies. We will also share how you can prevent these cookies from being stored however this may downgrade or 'break' certain elements of the sites functionality.<3/><4/>For more general information on cookies see the Wikipedia article on HTTP Cookies.<5/><6/><7>How We Use Cookies</7>": `
+      <0>What Are Cookies<0>
+      </1>
+      <2/>
+      As is common practice with almost all professional websites this site uses cookies, which are tiny files that are downloaded to your computer, to improve your experience. This page describes what information they gather, how we use it and why we sometimes need to store these cookies. We will also share how you can prevent these cookies from being stored however this may downgrade or 'break' certain elements of the sites functionality.
+      <3/>
+      <4/>
+      For more general information on cookies see the Wikipedia article on HTTP Cookies.
+      <5/>
+      <6/>
+      <7>How We Use Cookies</7>
+      `,
```

Comment:
@crysfel Do you like this approach ?
I feel it's little ugly 😞

Reply:
Given that the content has formatting and styles, I was thinking that maybe instead of putting all that in a the i18n files, maybe just creating `markdown` files that we can load dynamically and parse when needed.

After all people is not going to open this very often and it will increase our bundle size and is adding some unnecessary complexity to our codebase. 

Anyway, we can iterate on this. Let's create a separate issue for this and merge the PR as it is, it doesn't look great but we need to deliver something soon :)

Reply:
Yes, I agree with the markdown. It's something I commented below. It's going to be easier for future static pages. I would leave this how it's now and then.. have something like `markdown->JSX` converter.